src/Components/ModuleForm/ModuleForm.css

```diff
@@ -0,0 +1,128 @@
+.button-enabled {
+  color: white;
+  border: none;
+  border-radius: 10px;
+  padding: 1%;
+  height: 2%;
+  margin-left: 5%;
```

Comment:
I would avoid percentages for these properties. What is a 5% margin for example or a 1% padding? This doesn't really make sense and might have odd behaviour